modules/extensions-spring-support/src/main/java/org/mule/runtime/module/extension/internal/capability/xml/schema/builder/ObjectTypeSchemaDelegate.java

```diff
@@ -410,9 +413,27 @@ private TopLevelElement registerAbstractElement(QName typeQName, DslElementSynta
     return abstractElement;
   }
 
+  private Optional<QName> resolveSubstitutionGroupFromString(String userConfiguredSubstitutionGroup) {
+    String[] splittedSubstitutionGroup = userConfiguredSubstitutionGroup.split(":");
+    if (splittedSubstitutionGroup.length == 2) {
+      String namespacePrefix = splittedSubstitutionGroup[0];
+      String substitutionComponent = splittedSubstitutionGroup[1];
+      String namespaceUri = builder.getNamespaceUri(namespacePrefix);
+      if (namespaceUri != null) {
+        return Optional.of(new QName(namespaceUri, substitutionComponent, namespacePrefix));
+      }
```

Comment:
Means that the prefix is neither "mule" nor "ee" and that it's not within the imported namespaces.